pkgs/applications/altcoins/peercoin.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, autoreconfHook
+, openssl, db48, boost, zlib, miniupnpc
+, protobuf, utillinux, qt4, qrencode
```

Comment:
I think we should use qt5 if possible. Qt4 is soon to be deprecated.

Reply:
let me check if that even compiles

Reply:
Seems like most recent release supports qt5 from the .pro file